src/Runner.js

```diff
@@ -172,16 +161,63 @@ function run(transformFile, paths, options) {
         reject(e.message);
       });
     });
-  } else if (!fs.existsSync(transformFile)) {
-    process.stderr.write(
-      colors.white.bgRed('ERROR') + ' Transform file ' + transformFile + ' does not exist \n'
-    );
-    return;
-  } else {
-    return transform(transformFile);
+  } else if (fs.existsSync(transform)) {
+    let stats = fs.lstatSync(transform);
+
+    if (stats && stats.isFile(transform)) {
+      // handle file
+      return transform;
+    } else if (stats && stats.isDirectory(transform)) {
+      let directory = path.join(transform, 'transforms');
+      let file = path.join(transform, 'transformjs');
+
+      // handle directory
+      if (fs.existsSync(directory) && fs.lstatSync(directory).isDirectory()) {
+        return fs.readdirSync(directory);
+      } else if (fs.existsSync(file) && fs.lstatSync(file).isFile()) {
+        return file;
+      } else {
+        return fs.readdirSync(transform);
+      }
+    } else {
+      // handle other fs objects
+      return transform;
```

Comment:
Maybe it would be better to throw an error here? In which cases would this code path be hit?